pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "ansys-sherlock-core"
-version = "0.3.dev0"
+version = "0.9.dev0"
```

Comment:
This version hasn't been incremented properly. I bumped it up to match the version of the next release after 0.8, which is the 25.1 release.